addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiAqaraSensorMagnetHandler.java

```diff
@@ -0,0 +1,97 @@
+package org.openhab.binding.mihome.handler;
+
+import static org.openhab.binding.mihome.XiaomiGatewayBindingConstants.*;
+
+import org.eclipse.smarthome.core.library.types.DateTimeType;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.OpenClosedType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.UnDefType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.JsonObject;
+
+public class XiaomiAqaraSensorMagnetHandler extends XiaomiSensorBaseHandlerWithTimer {
+
+    private static final int DEFAULT_TIMER = 300;
+    private static final int MIN_TIMER = 30;
+    private static final String VOLTAGE = "voltage";
+    private static final String OPEN = "open";
+    private static final String CLOSED = "closed";
+    private static final String STATUS = "status";
+
+    private final Logger logger = LoggerFactory.getLogger(XiaomiSensorMagnetHandler.class);
+
+    public XiaomiAqaraSensorMagnetHandler(Thing thing) {
+        super(thing, DEFAULT_TIMER, MIN_TIMER, CHANNEL_OPEN_ALARM_TIMER);
+    }
+
+    @Override
+    void execute(ChannelUID channelUID, Command command) {
+        if (CHANNEL_OPEN_ALARM_TIMER.equals(channelUID.getId())) {
+            if (command != null && command instanceof DecimalType) {
+                setTimerFromDecimalType((DecimalType) command);
+                return;
+            }
+            // Only gets here, if no condition was met
+            logger.error("Can't handle command {} on channel {}", command, channelUID);
```

Comment:
In [E.4 of the coding guidelines](http://docs.openhab.org/developers/development/guidelines.html#e-logging) we listed what kind of logging level should be used when. Error should only be used:

> `error` logging should only be used to inform the user that something is tremendously wrong in his setup, the system cannot function normally anymore, and there is a need for immediate action. 

Please change this to use the correct logging levels.